packages/core/js/layout.ts

```diff
@@ -1,9 +1,7 @@
 export default {
-  spacingXXSmall: '4px',
-  spacingXSmall: '8px',
-  spacingSmall: '12px',
-  spacingMedium: '16px',
-  spacingLarge: '24px',
-  spacingXLarge: '48px',
-  spacingXXLarge: '64px'
+  main: '0',
+  globalSidenav: '930',
+  globalTopnav: '950',
+  globalDropdown: '1000',
+  globalSkiptoContentBanner: '1600'
 }
```

Comment:
i don't think we should be overwriting the layout file. guessing you mistook layout for layers

Reply:
lol, yeah, oops.  Will fix.